### PR TITLE
Local user login

### PR DESCRIPTION
**Pull request by sameetandpotatoes**
- Add /users/session route to stub out crowd and return a stubbed fake user

### CODE REVIEW

**Comment by sameetandpotatoes** [comment]
@aashishkapur @bcongdon This is ready for review

File: routes/users.rb [hunk 1]
```diff
@@ -76,19 +81,29 @@
...
-        # Should be mostly for development, where we don't have real user data
-        user_data = Auth.get_user_info(session_token)
-
-        user = User.create(
-            netid: netid,
-            first_name: user_data["user"]["first-name"],
-            last_name: user_data["user"]["last-name"],
-            is_member: true # must be an ACM user to have gotten here
-        )
+    if result["user"].to_json == local_fake_user.to_json
```
Comment (narendasan) [inline routes/users.rb]: This seems really weird to have fake user stubs in production code, is there not a way to have a stub service and add it to the dev manifest? Also what happens if someone uses these creds, they are public

Comment (narendasan) [inline routes/users.rb]: you could even fork this repo and pull out all the non auth stuff

Comment (narendasan) [inline routes/users.rb]: Was this code ever removed?

Comment (sameetandpotatoes) [inline routes/users.rb]: Will remove once I make merge the auth service stub integration PRs.

**Pull request merged**